mdnsctl/parser.c

```diff
@@ -448,6 +473,10 @@ parse_flags(const char *word, int *flags)
 			*flags |= F_A;
 			r++;
 			break;
+		case '6':
```

Comment:
The usage info for the "lookup" subcommand wasn't updated to include the new "6" flag.